histogramvec/histogramvec_test.go

```diff
@@ -344,11 +346,11 @@ func Test_Concurrency(t *testing.T) {
 	var wg sync.WaitGroup
 	for i := 0; i < 1000; i++ {
 		wg.Add(1)
-		go func() {
```

Comment:
Just curious if you remember what the error was here? It seems like this makes it not concurrent anymore

Reply:
The issue is that `t.Fatal` should not be called in the routine.

Reply:
I see, but this then is no longer concurrent. Maybe [passing the error out on a channel](https://github.com/ipfs/go-ipfs/issues/2043#issuecomment-164136026) would be an option?